examples/ili9341/pyportal_boing/feather-stm32f405.go

```diff
@@ -0,0 +1,34 @@
+// +build feather_stm32f405
+
+package main
+
+import (
+	"machine"
+
+	"tinygo.org/x/drivers/ili9341"
+)
+
+var (
+	csPin   = machine.D12
+	dcPin   = machine.D10
+	display = ili9341.NewSPI(
+		machine.SPI0,
+		dcPin,
+		csPin,
+		machine.D8, // if wired to 3.3V, pick an unused pin
+	)
+
+	// ILI9341's LED pin. set this to an unused pin (but not NoPin!) if
+	// wired via resistor straight to 3.3V. the boing example tries to
+	// set this pin and will panic if NoPin is used.
+	backlight = machine.D13
```

Comment:
I hadn't noticed.
It is better to have the following changes made.

```go
// examples/pyportal_boing/main.go - main()
	if backlight != machine.NoPin {
		// configure backlight
		backlight.Configure(machine.PinConfig{machine.PinOutput})
		backlight.High()
	}
```

I think the target source code is as follows.

* examples/ili9341/basic/main.go
* examples/ili9341/pyportal_boing/main.go
* examples/ili9341/scroll/main.go